appveyor.yml

```diff
@@ -1,9 +1,14 @@
-version: 0.5.{build}
-image:
-- Visual Studio 2017
-- Visual Studio 2015
-build_script:
-- cmd: >-
-    cmake .

```

Comment:
I imagine the line endings changed here?

Reply:
yes, i didn't notice they were wrong in the first place :(.

You can add [`?w=1`](https://github.com/nfrechette/acl/pull/17/files?w=1) to GitHub's diff page to see past the whitespace.